ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -102,6 +102,70 @@ def _get_table_description(schema: str, table: str, inspector: Inspector) -> str
     return description
 
 
+def get_dbt_details(config):
```

Comment:
Let's break this function down into smaller units:
- get dbt local
- get dbt s3
- get dbt gcs

Reply:
Yes